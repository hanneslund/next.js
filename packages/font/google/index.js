module.exports = require('../dist/google')
