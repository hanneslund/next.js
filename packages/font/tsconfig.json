{
  "compilerOptions": {
    "resolveJsonModule": true,
    "strict": true,
    "esModuleInterop": true,
    "outDir": "google",
    "module": "commonjs",
    "target": "es2019",
    "downlevelIteration": true,
    "preserveWatchOutput": true
  },
  "include": ["src/google/**/*.ts"],
  "exclude": ["node_modules"]
}
