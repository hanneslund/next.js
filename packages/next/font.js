module.exports = require('./dist/shared/lib/font')
