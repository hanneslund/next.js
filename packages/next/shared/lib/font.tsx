export function preloadFont(font: string) {
  console.log('PRELOADING: ' + font)
}
