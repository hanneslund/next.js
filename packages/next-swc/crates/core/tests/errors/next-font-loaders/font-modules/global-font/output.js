import '@next/font/google?{"font":"Inter","variant":"900"}'
import React from 'react'
