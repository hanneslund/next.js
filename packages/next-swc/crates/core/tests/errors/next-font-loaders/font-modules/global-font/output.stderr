
  x With font modules enabled you must assign the returned value to a const
   ,-[input.js:4:1]
 4 | ,-> Inter({
 5 | |     variant: '900',
 6 | `-> })
   `----
